bootstrap.sh

```diff
@@ -4,9 +4,13 @@ set -eu
 ## Make sure we have Ruby 3 installed directly on the instance
 amazon-linux-extras install -y ruby3.0
 
-## Install Terraform
-curl "https://releases.hashicorp.com/terraform/1.2.7/terraform_1.2.7_linux_amd64.zip" -o "terraform.zip"
-sudo unzip ./terraform.zip -d /usr/local/bin
+# Install Kubectl and Helm CLIs
+wget -q https://s3.us-west-2.amazonaws.com/amazon-eks/1.23.17/2023-05-11/bin/linux/amd64/kubectl -O /usr/local/bin/kubectl
```

Comment:
Can we install this directly from kubernetes instead of aws?

Reply:
That makes sense. I was following these [instructions](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) that AWS provides.